backend/app/utils/auth.py

```
@@ -0,0 +1,51 @@
+from typing import Optional
+
+import jwt
+from fastapi import Depends, HTTPException, status
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from passlib.context import CryptContext
+
+JWT_SECRET = os.getenv("JWT_SECRET", "change-me")
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES", "120"))
+
+from passlib.context import CryptContext
```

Ruff (0.14.2)

Redefinition of unused `CryptContext` from line 8

Remove definition: `CryptContext`

(F811)